src/workerd/io/actor-cache.c++

```diff
@@ -2581,7 +2582,10 @@ kj::Promise<void> ActorCache::flushImplUsingSinglePut(PutFlush putFlush) {
   // See the comment in flushImplUsingTxn for why we need to construct our RPC and then wait on
   // reads before actually sending the write. The same exact logic applies here.
   co_await waitForPastReads();
-  co_await request.send().ignoreResult();
+  {
+    util::DurationExceededLogger logger(clock, kj::SECONDS*1, "storage operation took longer than expected: single put");
```

Comment:
nit: in this file, `1 * kj::SECONDS` is much easier to read than `kj::SECONDS * 1`.